SR2022/2022-04-26-competition-wrap-up.md

```diff
@@ -0,0 +1,61 @@
+| 1st Place             | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk/)
+| 2nd Place             | [Queen Mary's College](https://www.qmc.ac.uk/)
+| 3rd Place             | [Brockenhurst College](https://brock.org.uk/)
+| Rookie Award          | [Abingdon School](https://www.abingdon.org.uk/)
+| Committee Award       | [Queen Mary's College](https://www.qmc.ac.uk/)
+| Robot and Team Image  | [Hills Road Sixth Form College](https://www.hillsroad.ac.uk/)
+| Online Presence       | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+
+## Event Media
+
+If you'd like some mementos of the competition or would just like to see the photos which were taken, we've collected them all together into [this photo album](TODO).
+
+The livestream of the matches also remains available on YouTube, as well as footage from the final:
+
+- Day 1: https://www.youtube.com/watch?v=7HNG2bqwXlo
+- Day 2: https://www.youtube.com/watch?v=M3stcLPbHOU
+
+## Feedback
+
+We'd love to get feedback on how the competition was for your competitors this year, so we know what we did well, and where we can improve. Please share the below short, anonymous survey with your competitors.
+
+[Send us feedback](https://docs.google.com/forms/d/e/1FAIpQLSeQXNw6XnATZM2LaBORIt_TgRqed3hNxpd2_2vdXXNxLY8ubA/viewform)
+
+Additionally, we welcome feedback from you the supervisors too. Please
+
+[Supervisor Feedback](https://docs.google.com/forms/d/e/1FAIpQLSc63utOabO5oBPOqgCZg-t1flOn1i5DzVC6GHcPXHEQ7_YDNg/viewform)
+
+Without all of your invaluable feedback we wouldn't be able to run the competition and attract new sponsors.
+
+## Outstanding kit
+
+Whilst most of you will have returned your kit at the end of the competition event, you may still have the other pieces of Student Robotics kit around your college. If you were unable to attend the competition, this will likely be a full kit.
+
+If you do still have some of our kit, please let us know as soon as possible so we can organise its safe return. It is vitally important that all kit is returned to us so we can clean, test and re-issue it fur future competitions.
+
+We will be contacting you soon if our records indicate you still have some items of kit.
+
+## Future Events
+
+We hope to run a competition next year and see you all there. We plan to open places around September and will let you know as soon as they are.
+
+In the meantime, be sure to follow us on Twitter [@StudentRobotics](https://twitter.com/studentrobotics).
+
+### Volunteering
+
+Student Robotics is powered by a collection of passionate individuals with varying skills from diverse backgrounds, many of whom are former competitors. We're all volunteers who give our spare time to make these events possible. If you have students graduating this year who may be interested in volunteering, please ask them to [sign up to be a blueshirt](https://studentrobotics.org/volunteer/).
```

Comment:
Is this the right url? I'd thought we were going to send competitors to the form at https://srobo.org/join, which handles some of them being under 18 currently?

Reply:
Looks like that URL is a much more direct CTA than the one on the website, so yes we should send them there.